draft-edm-protocol-greasing.md

```diff
@@ -37,40 +37,56 @@ informative:
 
 --- abstract
 
-Long-term interoperability of protocols is an important goal of the network
-standards process. Deployment success can depend on supporting change, which
-can include modifying how the protocol is used, extending the protocol, or
-replacing the protocol. This document presents concepts, considerations, and
-techniques related to protocol maintenance, such as greasing or variability. The
-intended audience is protocol designers and implementers.
+Active use and maintenance of network protocols is an important way to
+ensure that protocols remain interoperable and extensible over time.
+Techniques such as intentionally exercising extension points with
+non-meaningful values (referred to as "grease") or adding variability
+to how protocol elements are used help generate this active use.
 
+Grease and variability are used across various protocols developed
+by the IETF. This document discusses considerations when designing
+and deploying grease and variability mechanism, and provides
+advice for making them as effective as possible
```

Comment:
```suggestion
advice for making them as effective as possible.
```

Reply:
I like this new abstract, it is snappier. I seem to have a vague memory we included the target audience for a reason - but I can live with or without it here since its now in the intro.